### DISCUSSION TITLE
User roles are not working

### DISCUSSION THREAD

**Discussion by KernOil** [Budibase/budibase#2177] [Help] [answered]
Hi Guys,
after the updates from yesterday, none of my user roles are working anymore they all say undefined. 
I know this was mentioned under issues yesterday however I can not find the post anymore. just updated to 0.9.86 and am still having the same issue. any plan on a quick fix for this?  or do I need to define my own roles somehow ? 
![image](https://user-images.githubusercontent.com/80058238/127516694-33e4c740-ae18-4129-ab01-18ee486b010b.png)

**Comment by shogunpurple** [comment, marked as answer]
Hey @AndrewJRamay  - I noticed this one myself and then refreshed the page, and now I cannot recreate it on any of my environments. If a refresh does not work, would you be able to show me the output of the browser console when this issue occurs? Thanks!

**Reply by KernOil** [reply]
@shogunpurple the refresh overnight must have done the trick thank you.

**Reply by shogunpurple** [reply]
Thanks @AndrewJRamay - I will keep an eye on it because it happened to me and it was odd. Thanks for the reply!

**Comment by hhmmll** [comment]
the same issue also appear in my env, I tried to refresh but still not work for me
<img width="442" alt="image" src="https://user-images.githubusercontent.com/19184458/166632276-e468c9b7-13cc-4bc3-b2fd-d48779958856.png">
from the dev tool I can see the response
<img width="416" alt="image" src="https://user-images.githubusercontent.com/19184458/166632361-e4985cc4-d592-40e8-9e56-b8c212fc2f3a.png">
however it just  not appear on the option
